blacklist-control-plan.md

```diff
@@ -17,7 +17,7 @@ category: performance
 |**优化规则**|**规则名称**|**简介**|
```

Comment:
The file name of this file is `blocklist-control-plan.md`, added in https://github.com/pingcap/docs/pull/3043. I'll rename it later to keep zh and en consistent for easy maintenance.

Reply:
Got it. Thanks!